tests/utils/test_custom_button.py

```diff
@@ -0,0 +1,57 @@
+"""Tests for tasks related to loading and using csutom button yaml files"""
+import pytest
+from tamr_toolbox.utils.custom_button import (
+    create_redirect_button,
+    create_post_button,
+    delete_buttons,
+)
+import tempfile
+from pathlib import Path
+
```

Comment:
I see you have tested whether the exceptions you built in are being caught with these tests. But is there a way to test that button creation and function are working as intended? Might need input from other team members on this one...

Reply:
One obvious addition would be to add a yaml file, check that the file exists and is in the format that is expected. But the other stuff may not be possible since it involves a restart of Tamr... Maybe you could model it based on this? https://github.com/Datatamer/tamr-toolbox/blob/develop/tests/sysadmin/test_instance.py#L269

Reply:
I have personally tested the end to end flow using all the functions on a sandbox and they work as expected. I spoke with Keziah (who made the `test_instance.py` tests and she explained that they do not actually test that the functions work as expected since they require running the `unify-admin.sh` tool and restarting tamr etc. She explained it would be too much work to properly test this. However I have thoroughly tested on an internal instance manually